html-css/week1/Nina/order-desktop/order.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" type="text/css" href="order.css" />
+  <title>PAYMENT <span>FORM</span></title>
+</head>
+
+<body>
+  <header>
+    <h1>Order Payment</h1>
+    <h4>Please choose below</h4>
+  </header>
+  <!-- container -->
+  <main class="container">
+    <!-- Ordinary summary -->
+    <section>
+      <div class="ordinary-summary">Order Summary</div>
```

Comment:
```suggestion
      <h2 class="ordinary-summary">Order Summary</h2>
```
Avoid wrapping text only in `<div>` as it's missing it's semantic. It can be either a heading, paragraph, or `<span>` maybe when wrapped inside one of the other semantic content tags.